build-configs/configs/integreat-test-cms/assets/manifest.json

```diff
@@ -1 +1 @@
-../../integreat/assets/manifest.json
\ No newline at end of file
+'../../integreat/assets/manifest.json'
```

Comment:
Are you sure this is correct? This is a symlink and does reference the integreat manifest.json.